.github/workflows/triage.yml

```diff
@@ -22,6 +27,26 @@ jobs:
           name: event_payload
           path: ${{ github.event_path }}
 
+  workflows-label:
+    if: always() && github.repository_owner == 'Homebrew'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check pull request changed files
```

Comment:
I should rewrite this in JavaScript (ugh), since actions/github-script makes it really easy to [retry](https://github.com/actions/github-script#retries) API calls.

Reply:
Looks like missing PAT for this job.

Reply:
> Looks like missing PAT for this job.

#132649